halyard-core/src/main/java/com/netflix/spinnaker/halyard/core/registry/v1/GoogleProfileReader.java

```diff
@@ -112,7 +102,39 @@ String bomPath(String version) {
   private InputStream getContents(String objectName) throws IOException {
     ByteArrayOutputStream output = new ByteArrayOutputStream();
     log.info("Getting object contents of " + objectName);
-    googleStorage.objects().get(spinconfigBucket, objectName).executeMediaAndDownloadTo(output);
+
+    try {
+      applicationDefaultGoogleStorage.objects().get(spinconfigBucket, objectName).executeMediaAndDownloadTo(output);
+    } catch (IOException e) {
+      log.debug("Getting object contents of {} with failed. Retrying with no authentication.", objectName, e);
```

Comment:
I think there is an extraneous 'with' in this msg.

Reply:
Thanks! Fixed in #930